scripts/benchmarks/format-benchmark-record.js

```diff
@@ -13,14 +13,10 @@ const Constants = require('../../constants.js');
  * @enum {string}
  */
 const SUBMISSION_METHOD_MAP = {
-  'claims': 'claims',
-  'registry': 'registry',
-  'registry/qcdr': 'registry',
-  'cmswebinterface': 'cmsWebInterface',
-  'administrativeclaims': 'administrativeClaims',
-  'ehr': 'electronicHealthRecord',
-  'cmsapprovedcahpsvendor': 'certifiedSurveyVendor',
-  'cahpssurveyvendor': 'certifiedSurveyVendor'
+  'ecqm': 'electronicHealthRecord',
+  'medicarepartbclaims': 'claims',
+  'mipscqm': 'registry',
+  'qcdrmeasure': 'registry'
```

Comment:
I suppose this is fine, but I wanted to check

Reply:
All of them got renamed, see https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPARCH&title=Task%3A+Aggregate+Multiple+Submissions+and+Update+Terminology

Beyond that, it removes spaces + lower-cases everything so the keys look a bit weird. Kept the logic alone for now and just had the weird keys.